src/libs/OptionsListUtils.ts

```diff
@@ -16,47 +24,97 @@ import * as LoginUtils from './LoginUtils';
 import Navigation from './Navigation/Navigation';
 import Permissions from './Permissions';
 import * as PersonalDetailsUtils from './PersonalDetailsUtils';
+import {PersonalDetailsList} from './PolicyUtils';
 import * as ReportActionUtils from './ReportActionsUtils';
 import * as ReportUtils from './ReportUtils';
 import * as TaskUtils from './TaskUtils';
 import * as TransactionUtils from './TransactionUtils';
 import * as UserUtils from './UserUtils';
 
+type Tag = {enabled: boolean; name: string; accountID: number | null};
+
+type Option = {text: string; keyForList: string; searchText: string; tooltipText: string; isDisabled: boolean};
+
+type PayeePersonalDetails = {text: string; alternateText: string; icons: OnyxCommon.Icon[]; descriptiveText: string; login: string; accountID: number};
+
+type CategorySection = {title: string | undefined; shouldShow: boolean; indexOffset: number; data: Option[] | Participant[] | ReportUtils.OptionData[]};
```

Comment:
```suggestion
type CategorySection = {title?: string; shouldShow: boolean; indexOffset: number; data: Option[] | Participant[] | ReportUtils.OptionData[]};
```

Consider using `?` instead of `undefined`

Reply:
This is something different , when using `?` we can not pass a title property but we want always it to be passed but it can be undefined